src/components/FormInput/style.ts

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { Color, fonts } from "../../ui";
+
+interface InputStyleProps {
+  error: boolean;
```

Comment:
it is better to name this var `isError` or something like this because just `error` would mean that you have a string or an object